test-cases/tests/backup-restore/j05-verify-3scale-backup-and-restore.md

```diff
@@ -18,9 +18,27 @@ Note: this test should only be performed at a time it will not affect other ongo
 
 ## Steps
 
-### Postgres
+### Pre-requisites
 
-1. Verify data exists in postgres using the terminal in the `standard-auth` pod in the `redhat-rhmi-operator` namespace
+1. Create a throw-away Postgres instance
+
+```sh
+cat << EOF | oc create -f - -n redhat-rhmi-operator
+  apiVersion: integreatly.org/v1alpha1
+  kind: Postgres
+  metadata:
+    name: throw-away-postgres
+    labels:
+      productName: productName
+  spec:
+    secretRef:
+      name: throw-away-postgres-sec
+    tier: development
+    type: workshop
+EOF
+```
+
+- Run the following commands in throwaway Postgres pod and verify that data in 3Scale Postgres exists:
```

Comment:
Hey @philbrookes, thank you. This throw away pod is only to keep consistent with what was being checked before, in managed installation we have used one of AMQ pods to connect to 3Scale Postgres. Since we don't have AMQ anymore I thought it would be good to create one just to confirm data is there just like we did before. The second throwaway pod is used to connect to 3Scale Postgres to create a dump file. So the first step in this test would stay as it was before, while the next two steps are automated.